data/assets.gresource.xml

```diff
@@ -47,6 +47,6 @@
     <file alias="shape-circle-symbolic.svg">icons/symbolic/shape-circle-symbolic.svg</file>
     <file alias="shape-all-symbolic.svg">icons/symbolic/shape-all-symbolic.svg</file>
     <file alias="shape-text-symbolic.svg">icons/symbolic/shape-text-symbolic.svg</file>
-    
+
```

Comment:
Nitpick, but please revert this file :) It helps to touch as little files as possible to keep the commit and blame history as clean as possible

Reply:
Reverted with the help of @albfan!